<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登入</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_shadow" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_back" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_frame" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_pop_cont" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,1">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>18</number>
                 </property>
                 <property name="leftMargin">
                  <number>33</number>
                 </property>
                 <property name="topMargin">
                  <number>30</number>
                 </property>
                 <property name="rightMargin">
                  <number>33</number>
                 </property>
                 <property name="bottomMargin">
                  <number>18</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_name">
                     <property name="minimumSize">
                      <size>
                       <width>60</width>
                       <height>24</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>24</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>用户:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_name">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>29</height>
                      </size>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Normal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_password">
                     <property name="minimumSize">
                      <size>
                       <width>60</width>
                       <height>24</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>60</width>
                       <height>24</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>密码:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_password">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>29</height>
                      </size>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Password</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>33</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_ok">
                   <property name="minimumSize">
                    <size>
                     <width>102</width>
                     <height>29</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>102</width>
                     <height>29</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>确定</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>14</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_cancel">
                   <property name="minimumSize">
                    <size>
                     <width>102</width>
                     <height>29</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>58</width>
                     <height>29</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>取消</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QWidget" name="widget_msg" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>30</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer_8">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_msg">
                    <property name="text">
                     <string>Messge</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <property name="margin">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_9">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>lineEdit_name</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>LoginDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
